src/pages/zh-cn/reference/errors/static-client-address-not-available.mdx

```diff
@@ -0,0 +1,19 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro.clientAddress is not available in static mode.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **StaticClientAddressNotAvailable**: `Astro.clientAddress` 仅在设置 `output: 'server'` 时可用. 如果您需要服务端渲染，请更新您的 Astro 配置。(E03003)
+
+## 哪里出了问题？
+只有在启用[服务端渲染](/zh-cn/guides/server-side-rendering/)时，`Astro.clientAddress` 属性才可用。
+
+如要在静态模式下获取用户的IP地址，您可以在[客户端脚本](/zh-cn/core-concepts/astro-components/#客户端脚本) 中使用类似于 [Ipify](https://www.ipify.org/) 等不同的API，或者您可以使用在服务提供商所托管的无服务器 (serverless) 函数来获取用户的IP。
```

Comment:
```suggestion
如要在静态模式下获取用户的IP地址，您可以在[客户端脚本](/zh-cn/core-concepts/astro-components/#客户端脚本) 中使用类似于 [Ipify](https://www.ipify.org/) 等不同的API，或者您可以使用托管在服务提供商处的无服务器 (serverless) 函数来获取用户的IP。
```

Reply:
Thanks for the review! I'll focus more on readability.